### PR TITLE
Cgat scripts

### PR DESCRIPTION
**Pull request by sebastian-luna-valero**
* [X] I have read the guidelines above.
* [X] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

@bioconda/core please review.

### CODE REVIEW

File: recipes/cgat-scripts/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,207 @@
+
+package:
+  name: cgat-scripts
+  version: 0.2.6
```
Comment (johanneskoester) [inline recipes/cgat-scripts/meta.yaml]: Can you please move the version to the top as a jinja variable, and use the non-hash URL of pypi below? See the snakemake recipe for an example.

File: recipes/cgat-scripts/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+build:
+  skip: True # [not py27 or osx]
+  number: 0
+  entry_points:
+    - cgat = CGAT.cgat:main
+
+requirements:
+  build:
+    # system deps
+    - nomkl
```
Comment (johanneskoester) [inline recipes/cgat-scripts/meta.yaml]: Can these pinnings be relaxed (e.g. to the major versions) or do you definitely want to keep them so strict?

File: recipes/cgat-scripts/meta.yaml [hunk 3]
```diff
@@ -1,10 +1,11 @@
+{% set version = "0.2.6" %}
 
 package:
   name: cgat-scripts
-  version: 0.2.6
+  version: {{ version }}
 
 source:
-  url: https://pypi.python.org/packages/fe/86/be2a14843dc272abe33dd0de754e8ba142fc07c3cfe11410be7230d27078/CGAT-0.2.6.tar.gz
+  url: https://pypi.io/packages/source/C/CGAT/CGAT-0.2.6.tar.gz
```
Comment (johanneskoester) [inline recipes/cgat-scripts/meta.yaml]: You can now reuse the version variable in the URL.

File: recipes/cgat-scripts/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,208 @@
+{% set version = "0.2.6" %}
+
+package:
+  name: cgat-scripts
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/C/CGAT/CGAT-0.2.6.tar.gz
```
Comment (johanneskoester) [inline recipes/cgat-scripts/meta.yaml]: You can now reuse the version variable in the URL.

**Comment by sebastian-luna-valero** [comment]
Tests passing, so will merge.

Thanks!

**Comment by sebastian-luna-valero** [comment]
Hello,

If I go the the website of this package:
http://bioconda.github.io/recipes/cgat-scripts/README.html

and follow the instructions in the biocontainers page:
https://quay.io/repository/biocontainers/cgat-scripts

I get:
```
docker pull quay.io/biocontainers/cgat-scripts
Using default tag: latest
Pulling repository quay.io/biocontainers/cgat-scripts
Tag latest not found in repository quay.io/biocontainers/cgat-scripts
``` 

Instead of the `latest` the tag there is the conda `buildstring`:
https://quay.io/repository/biocontainers/cgat-scripts?tab=tags

Is this intended?

Many thanks,
Sebastian

**Comment by johanneskoester** [comment]
@bgruening did the way of storing stuff in biocontainers change?

**Pull request merged**